src/genericmera.jl

```diff
@@ -513,12 +505,10 @@ an initial guess for the fixed-point density matrix.
 
 See also: [`fixedpoint_densitymatrix`](@ref)
 """
-function thermal_densitymatrix(m::GenericMERA{N, LT, OT}, depth) where {N, LT, OT}
+function thermal_densitymatrix(m::GenericMERA, depth)
     width = causal_cone_width(typeof(m))
-    V = inputspace(m, depth)^width
-    rho_tensor = id(Matrix{eltype(m)}, V)
-    rho_op::OT = convert(OT, rho_tensor)
-    return rho_op
+    V = ⊗(ntuple(n->inputspace(m, depth), Val(width))...)
+    return id(storagetype(operatortype(m)), V)
```

Comment:
Could this be wrapped in `convert(operatortype(m), ...)`?

Reply:
Sure, this anyway is currently broken for `ModifiedBinary...`